src/MainMazeClass.java

```diff
@@ -0,0 +1,26 @@
+import controller.MazeController;
+import model.MazeModel;
+import view.CLI;
+
+// Main class that initiating the main application:
+public class MainMazeClass {
+
+    public static void main(String[] args) throws Exception {
+
+        // Model:
+        MazeModel mazeModel = new MazeModel();
+
+        // View:
+        CLI mazeCli = new CLI();
+        // Instead of using CLI, we can use is.txt as input commands and os.txt as output:
+        // CLI _cli = new CLI("is.txt", "os.txt", mazeModel);
```

Comment:
Wow! Sounds like a **great** add-on
Just why is the model directly passed? (un-necessery coupling?)

Reply:
Removed, forgot to remove the passed parameter